homeassistant/components/pooldose/sensor.py

```diff
@@ -32,6 +37,17 @@
         device_class=SensorDeviceClass.VOLTAGE,
         native_unit_of_measurement=UnitOfElectricPotential.MILLIVOLT,
     ),
+    SensorEntityDescription(
+        key="cl",
+        translation_key="cl",
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+    ),
+    SensorEntityDescription(
+        key="flow_rate",
+        translation_key="flow_rate",
+        device_class=SensorDeviceClass.VOLUME_FLOW_RATE,
+        # Unit dynamically determined via API
```

Comment:
Please do note that all units need to be in the list of supported units for the device class to work

Reply:
checked, both available.
- LITERS_PER_SECOND = "L/s"
- CUBIC_METERS_PER_HOUR = "m³/h"

But I see m3/h (from pooldose) is not m³/h (in HA const.py:UnitOfVolumeFlowRate).. 

I need to convert this.

Reply:
Oh but you can also map in HA. Like ideally we use HA units as it can be that we sometimes change them for whatever reason (for example recently we round out that we were using different micro-symbols throughout the codebase and we wanted to make it consistent